packages/ember-glimmer/tests/integration/helpers/text-area-test.js

```diff
@@ -0,0 +1,58 @@
+import { set } from 'ember-metal/property_set';
+import TextArea from 'ember-views/views/text_area';
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+
+class TextAreaRenderingTest extends RenderingTest {
+  constructor() {
+    super();
+
+    this.registerComponent('-text-area', { ComponentClass: TextArea });
+    this.render('{{textarea disabled=disabled value=val}}', { val: 'Lorem ipsum dolor' });
+  }
+}
+
+moduleFor('Helpers test: {{textarea}}', class extends TextAreaRenderingTest {
+
+  ['@htmlbars Should insert a textarea']() {
+    equal(this.$('textarea').length, 1, 'There is a single textarea');
```

Comment:
So for this test, I think you would want to do something like this:

```javascript
['@htmlbars it can insert a basic textarea']() {
  this.render('{{textarea}}');

  equal(this.$('textarea').length, 1, 'There is a single textarea');

  this.assertStableRerender();
}
```